release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2999,6 +2999,10 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.13.4 --pullspecs
 ----
 
+[id="ocp-4-13-4-bug-fixes"]
+==== Bug fixes
+* Previously, you could not use persistent volume storage on a cluster with Confidential virtual machines (VMs). This issue persists with {product-title} 4.13.3 and earlier versions. From {product-title} 4.13.4 and later versions, you can now use persistent volume storage on a cluster with Confidential VMs. (link:https://issues.redhat.com/browse/OCPBUGS-11768[*OCPBUGS-11768*])
```

Comment:
You may want to mention GCP somewhere here, such as "on a cluster installed on Google Cloud Platform with Confidential virtual machines".

Reply:
Thanks, Ben. I went with:

> you could not use persistent volume storage on a cluster with Confidential virtual machines (VMs) on Google Cloud Platform (GCP)